tests/e2e/ledger/allow_addrs.bats

```diff
@@ -13,7 +13,7 @@ function setup() {
     if ! [ $CI ]; then
         (
           cd "$GIT_ROOT"
-          cargo build
+          cargo build --all-features
```

Comment:
Non blocking but can we also just `build` without features? I'd like to build every combinations of features but we aren't there yet. If you `build` first without features it will not be that costly to rebuild after.

Reply:
Not possible for this test nor for most of the resiliency tests since they rely on "*_testing" features.

Would you prefer we target the feature(s) required to run the test instead? We could also create a "testing" feature that would activate all the "*_testing".

WDYT?